### PR TITLE
chore: Add Node.js v17 to CI

### PR DESCRIPTION
**Pull request by rodion-arr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,6 +52,6 @@
   },
   "support": true,
   "engines": {
-    "node": "^16 || ^14 || ^12"
+    "node": "^17 || ^16 || ^14 || ^12"
```
Comment (dominykas) [inline package.json:55]: I think it's OK to add it to the test matrix, but I wouldn't advertise it as supported?

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [ 16, 14, 12 ]
+        node-version: [ 17, 16, 14, 12 ]
```
Comment (dominykas) [inline .github/workflows/ci.yaml:20]: Ideally we'd switch this to pkgjs/action, but there's no way to pass in the env vars just yet...

**Comment by dominykas** [comment]
Thanks for the reminder!

File: package.json (hunk 1 above)
Comment (rodion-arr) [reply package.json]: Agree, reverted

**Review by dominykas — APPROVED** [review]

**Pull request merged**